facebook/Module.md

```diff
@@ -67,161 +67,156 @@ facebook:FacebookConfiguration facebookConfig = {
 facebook:Client facebookclient = new(facebookConfig);
```

Comment:
Better to rename this `facebookclient` to `facebookClient` (including all the usages) as well.